### PR TITLE
Bugfix – Remove quotes in boolean JSON values in BioSolr config

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
Because in JavaScript/JSON `"false"` evaluates to `true`. Since this is a schema change I bumped the env file to reflect that.

I also took the opportunity to add another env file for `scxa-gene2experiment` and fix a few typos here and there.

### CODE REVIEW

File: bin/create-scxa-analytics-schema.sh [hunk 1]
```diff
@@ -376,10 +376,10 @@ HTTP_STATUS=$(curl $SOLR_AUTH -w "%{http_code}" -o >(cat >&3) -X POST -H 'Conten
     "class": "biosolr:uk.co.flax.biosolr.solr.update.processor.OntologyUpdateProcessorFactory",
     "annotationField": "ontology_annotation",
     "ontologyURI": "'$SCXA_ONTOLOGY'",
-    "includeChildren": "false",
-    "includeDescendants": "false"
+    "includeChildren": false,
+    "includeDescendants": false
```
Comment (pmb59) [inline bin/create-scxa-analytics-schema.sh:380]: same in L342 above?  (    `"update.autoCreateFields": "false"`)

**Review by pmb59 — APPROVED** [review]
LGTM @alfonsomunozpomer 

I've removed few other quotes in boolean JSON values here [34953af](https://github.com/ebi-gene-expression-group/index-scxa/pull/39/commits/34953af612b775fc5c24d0ec6b0a91c6ce15126c) Please check it before merge

**Comment by alfonsomunozpomer** [comment]
> LGTM @alfonsomunozpomer
> 
> I've removed few other quotes in boolean JSON values here [34953af](https://github.com/ebi-gene-expression-group/index-scxa/pull/39/commits/34953af612b775fc5c24d0ec6b0a91c6ce15126c) Please check it before merge

Good catch. I missed those. The fact that suggester and other fields behaved as expected makes me think that Solr parses the strings, but if that’s the case it’s undocumented behaviour and we shouldn’t rely on it. Thanks.

**Comment by alfonsomunozpomer** [comment]
In an unexpected turn of events, it turns out that Solr expects those boolean values to be quoted. So https://github.com/ebi-gene-expression-group/index-scxa/pull/39/commits/34953af612b775fc5c24d0ec6b0a91c6ce15126c should be reverted!

**Comment by pmb59** [comment]
> In an unexpected turn of events, it turns out that Solr expects those boolean values to be quoted. So [34953af](https://github.com/ebi-gene-expression-group/index-scxa/commit/34953af612b775fc5c24d0ec6b0a91c6ce15126c) should be reverted!

Only those in https://github.com/ebi-gene-expression-group/index-scxa/commit/34953af612b775fc5c24d0ec6b0a91c6ce15126c or all `false` in this PR?

**Comment by alfonsomunozpomer** [comment]
> Only those in [34953af](https://github.com/ebi-gene-expression-group/index-scxa/commit/34953af612b775fc5c24d0ec6b0a91c6ce15126c) or all `false` in this PR?

Only the last ones. The previous commit, https://github.com/ebi-gene-expression-group/index-scxa/pull/39/commits/e602015fad5921b02a48d1c5ed4842a443b8985e, had a green build.

**Comment by pmb59** [comment]
> > Only those in [34953af](https://github.com/ebi-gene-expression-group/index-scxa/commit/34953af612b775fc5c24d0ec6b0a91c6ce15126c) or all `false` in this PR?
> 
> Only the last ones. The previous commit, [e602015](https://github.com/ebi-gene-expression-group/index-scxa/commit/e602015fad5921b02a48d1c5ed4842a443b8985e), had a green build.

OK, I will revert

**Comment by pmb59** [comment]
> > > Only those in [34953af](https://github.com/ebi-gene-expression-group/index-scxa/commit/34953af612b775fc5c24d0ec6b0a91c6ce15126c) or all `false` in this PR?
> > 
> > 
> > Only the last ones. The previous commit, [e602015](https://github.com/ebi-gene-expression-group/index-scxa/commit/e602015fad5921b02a48d1c5ed4842a443b8985e), had a green build.
> 
> OK, I will revert

done - passing checks

**Review by irisdianauy — APPROVED** [review]

**Pull request merged**